uniffi_bindgen/src/lib.rs

```diff
@@ -230,15 +243,38 @@ pub fn run_tests<P: AsRef<Path>>(
     }
 
     for (lang, test_scripts) in language_tests {
-        bindings::write_bindings(&component, &cdylib_dir, lang, true)?;
-        bindings::compile_bindings(&component, &cdylib_dir, lang)?;
+        bindings::write_bindings(&config.bindings, &component, &cdylib_dir, lang, true)?;
+        bindings::compile_bindings(&config.bindings, &component, &cdylib_dir, lang)?;
         for test_script in test_scripts {
             bindings::run_script(cdylib_dir, &test_script, lang)?;
         }
     }
     Ok(())
 }
 
+fn get_config(component: &ComponentInterface) -> Result<Config> {
+    let default_config: Config = component.into();
+
+    let pkg_path: PathBuf = env::var("CARGO_MANIFEST_DIR")
+        .expect("Missing $CARGO_MANIFEST_DIR, cannot load config file")
+        .into();
+
+    let config_file = pkg_path.join("uniffi.toml").canonicalize();
+
+    if config_file.is_err() {
+        return Ok(default_config);
+    }
+
+    let config_file = config_file.unwrap();
+    let contents = slurp_file(&config_file)
+        .map_err(|_| anyhow!("Failed to read config file from {:?}", &config_file))?;
```

Comment:
Another small nit: I like using `.context()` and `.with_context()` (you might need to `use anyhow::Context` at the top) instead of `map_err`, like this...

```suggestion
    let contents = slurp_file(&config_file)
        .with_context(|| format!("Failed to read config file from {:?}", &config_file))?;
```

Because that preserves the original error, so you'll get, say, the syntax error in addition to the context of where it came from.